CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased] - 2019-07-12
+
+- Add Rails 6 support
+- Lock gems for Rails 5 specs
```

Comment:
I don't think this is useful to know from a Simple Token Authentication user point of view. It is very much a gem maintenance concern.

```suggestion
```